### ISSUE TITLE
Query on Evaluation Capabilities for Reasoning Chains in Embodied Intelligence Models

### ISSUE THREAD

**Issue by forevercarry** [embodied-agent-interface/embodied-agent-interface#7]
Thank you for your contributions to embodied intelligence. However, from the content of the article and the way it is invoked here, it seems to only support the evaluation and comparison of models, and does not support the evaluation and comparison of reasoning chains similar to ReAct. I'm not sure if my understanding is correct?

**Comment by liyichen-cly** [comment]
> Thank you for your contributions to embodied intelligence. However, from the content of the article and the way it is invoked here, it seems to only support the evaluation and comparison of models, and does not support the evaluation and comparison of reasoning chains similar to ReAct. I'm not sure if my understanding is correct?

I think it's like this. They require the generation of LLM outputs that conform to their defined specifications to be placed in the "llm-response-path", without including how to generate the llm response.

**Comment by bryanzhou008** [comment]
Yes, that is correct. If your reasoning chain is able to generate outputs in the same format then you can use our evaluation code and pipeline to get detailed metrics and feedback. Otherwise you can also have an automatic script translate the results of your method to our format for evaluation.
